servant-client/test/Servant/ClientSpec.hs

```diff
@@ -208,47 +208,48 @@ sucessSpec :: Spec
 sucessSpec = beforeAll (startWaiApp server) $ afterAll endWaiApp $ do
 
     it "Servant.API.Get" $ \(_, baseUrl) -> do
-      let getGet = getNth (Proxy :: Proxy 0) $ client api baseUrl manager
-      (left show <$> runExceptT getGet) `shouldReturn` Right alice
+      let getGet = getNth (Proxy :: Proxy 0) $ client api
+      (left show <$> SCR.runClientM getGet baseUrl manager) `shouldReturn` Right alice
```

Comment:
I'm hoping we can now get rid of the `getNth` hack and just declare `getGet`, `getDeleteEmpty`, etc. at the top level. (This shouldn't necessarily be part of this PR. Just saying that I like this change. :) )